docs/contributors/setup/set-up-git-for-ubuntu-desktop-work.md

```diff
@@ -0,0 +1,458 @@
+---
+relatedlinks: "[PackagingWithGit &#32; - &#32; Debian Wiki](https://wiki.debian.org/PackagingWithGit)"
+---
+
+(set-up-git-for-ubuntu-desktop-work)=
+# Set up Git for Ubuntu Desktop work
+
+The [Desktop Team](https://library.canonical.com/our-organisation/ubuntu-engineering/desktop) uses Git and Git Build Package (`gbp`) to maintain GNOME packages for Ubuntu. Let's set up your system so that you can contribute to GNOME on Ubuntu Desktop.
+
+We'll use the Settings application as an example. Internally, the application is known as the GNOME Control Center.
+
+
+## Initial setup
+
+1. Install the basic packages for working in Git and Ubuntu packaging:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir:
+    :user:
+    :input: sudo apt install git git-buildpackage ubuntu-dev-tools gnupg
+    ```
+
+    For details about the packaging environment, see [Getting Set Up](https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/1.0/tutorial/getting-set-up.html) in the Ubuntu Packaging Guide.
```

Comment:
The Packaging Guide is deprecated. Use resources from within these docs, please.
```suggestion
    For details about the packaging environment, see {ref}`how-to-set-up-for-ubuntu-development`.
```